go_sequencer/txprocessor/txprocessor.go

```diff
@@ -153,5 +177,1068 @@ func NewTxProcessor(state *statedb.StateDB, config Config) *TxProcessor {
 // created accounts.
 func (txProcessor *TxProcessor) ProcessTxs(coordIdxs []common.AccountIdx, l1usertxs, l1coordinatortxs []common.L1Tx,
 	l2txs []common.PoolL2Tx) (ptOut *ProcessTxOutput, err error) {
-	return nil, nil
+	defer func() {
+		if err == nil {
+			err = txProcessor.state.MakeCheckpoint()
+		}
+	}()
+
+	var exitTree *merkletree.MerkleTree
+	var createdAccounts []common.Account
+
+	if txProcessor.zki != nil {
+		return nil, common.Wrap(
+			errors.New("Expected StateDB.zki==nil, something went wrong and it's not empty"))
+	}
+	// defer txProcessor.resetZKInputs()
+
+	if len(coordIdxs) > int(txProcessor.config.MaxFeeTx) {
+		return nil, common.Wrap(
+			fmt.Errorf("CoordIdxs (%d) length must be smaller than MaxFeeTx (%d)",
+				len(coordIdxs), txProcessor.config.MaxFeeTx))
+	}
+
+	nTx := len(l1usertxs) + len(l1coordinatortxs) + len(l2txs)
```

Comment:
Also, Here @kaiden-araki Can you please explain a bit about the use of L1coordinatortxs in our implementation. Like what will we process, here ? Will this be required for us ?